### PR TITLE
MAB gliders added to vocab.csv

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added 3 MAB gliders for upcoming P20c cruise to vocab.csv (can compare to the same NES CP05MOAS gliders)

### CODE REVIEW

File: vocab/vocab.csv [hunk 1]
```diff
@@ -1216,20 +1216,41 @@ CP15MOAS-G0913-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 913
...
 CP15MOAS-G0913-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 913,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP15MOAS-GL339,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,,,,0,1000
+CP15MOAS-GL339-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP15MOAS-GL339-01-ADCPAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP15MOAS-GL339-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP15MOAS-GL339-03-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP15MOAS-GL339-04-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP15MOAS-GL339-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
```
Comment (reedan88) [inline vocab/vocab.csv:1225]: Looks good

File: vocab/vocab.csv [hunk 2]
```diff
@@ -1216,20 +1216,41 @@ CP15MOAS-G0913-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 913
...
 CP15MOAS-GL379-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 379,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP15MOAS-GL380,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,,,,0,1000
+CP15MOAS-GL380-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP15MOAS-GL380-01-ADCPAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP15MOAS-GL380-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP15MOAS-GL380-03-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP15MOAS-GL380-04-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP15MOAS-GL380-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
```
Comment (reedan88) [inline vocab/vocab.csv:1239]: Looks good.

File: vocab/vocab.csv [hunk 3]
```diff
@@ -1216,20 +1216,41 @@ CP15MOAS-G0913-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 913
...
 CP15MOAS-GL387-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP15MOAS-GL559,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,,,,0,1000
+CP15MOAS-GL559-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP15MOAS-GL559-01-ADCPAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP15MOAS-GL559-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP15MOAS-GL559-03-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP15MOAS-GL559-04-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP15MOAS-GL559-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
```
Comment (reedan88) [inline vocab/vocab.csv:1253]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good. Only one question: the range for 339 changed from 200 to 1000 correct?

File: vocab/vocab.csv [hunk 4]
```diff
@@ -1216,20 +1216,41 @@ CP15MOAS-G0913-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 913
...
 CP15MOAS-G0913-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 913,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP15MOAS-GL339,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,,,,0,200
+CP15MOAS-GL339-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,200
+CP15MOAS-GL339-01-ADCPAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,200
+CP15MOAS-GL339-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,200
+CP15MOAS-GL339-03-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,200
+CP15MOAS-GL339-04-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,Dissolved Oxygen,Aanderaa,Optode 4831,0,200
+CP15MOAS-GL339-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 339,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
```
Comment (cdobs) [inline vocab/vocab.csv:1225]: looks good

File: vocab/vocab.csv [hunk 5]
```diff
@@ -1216,20 +1216,41 @@ CP15MOAS-G0913-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 913
...
 CP15MOAS-GL379-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 379,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP15MOAS-GL380,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,,,,0,1000
+CP15MOAS-GL380-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP15MOAS-GL380-01-ADCPAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP15MOAS-GL380-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP15MOAS-GL380-03-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP15MOAS-GL380-04-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP15MOAS-GL380-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 380,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
```
Comment (cdobs) [inline vocab/vocab.csv:1239]: looks good

File: vocab/vocab.csv [hunk 6]
```diff
@@ -1216,20 +1216,41 @@ CP15MOAS-G0913-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 913
...
 CP15MOAS-GL387-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP15MOAS-GL559,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,,,,0,1000
+CP15MOAS-GL559-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP15MOAS-GL559-01-ADCPAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP15MOAS-GL559-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP15MOAS-GL559-03-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP15MOAS-GL559-04-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP15MOAS-GL559-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 559,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
```
Comment (cdobs) [inline vocab/vocab.csv:1253]: looks good

**Review by cdobs — APPROVED** [review]

File: vocab/vocab.csv (hunk 4 above)
Comment (reedan88) [inline vocab/vocab.csv:1225]: Looks good

**Review by reedan88 — APPROVED** [review]
Reviewed changes and they look good.

**Pull request merged**